source/includes/extracts-download-from-tools.yaml

```diff
@@ -0,0 +1,101 @@
+ref: _download-from-tools
+content: |
+  The {{command}} tool is part of the MongoDB tools package. Consult the
```

Comment:
Small warning - it appears that for the latest 4.3 dev series there's just no package for the tools. Or at least not one I Can find. hopefully they resolve that soon!